.ci/README.md

```diff
@@ -0,0 +1,179 @@
+# CI documentation
+
+This document pretends to clarify the CI status and configuration for the RHBA set of projects. [See projects dependency tree](https://github.com/kiegroup/droolsjbpm-build-bootstrap#kiegroup-project-structure).
+
+Table of content
+----------------
+
+* **[We use cross-repo pull requests as CI strategy](#we-use-cross-repo-pull-requests-as-ci-strategy)**
+
+* **[Where are the jobs defined?](#where-are-the-jobs-defined)**
+
+* **[Where can I find the commands which are going to be executed?](#where-can-i-find-the-commands-which-are-going-to-be-executed)**
+
+* **[How are the projects related between them?](#how-are-the-projects-related-between-them)**
```

Comment:
is "related" proper word/term? isn't better to use "depends" or "depending on each other"

Reply:
```suggestion
* **[How are the project dependencies defined?](#how-are-the-project-dependencies-defined)**
```
wdyt @mareknovotny ?